javascript/Makefile

```diff
@@ -55,6 +55,9 @@ dist/gherkin.js: lib/gherkin/parser.js ../LICENSE node_modules/.fetched
 	echo '*/' >> $@
 	./node_modules/.bin/browserify index.js --ignore-missing >> $@
 
+ace: lib/gherkin/parser.js ../LICENSE node_modules/.fetched
```

Comment:
The name of the target should not be `ace`, but the path to the file that gulp produces. Otherwise, `make` won't be able to detect that there is nothing to do.